### PR TITLE
Updated README.md

### PR DESCRIPTION
**Pull request by muskanvk**
#### References to other Issues or PRs

Updated the README.md file. Added links for stackoverflow search with sympy tag.

#### Release Notes

NO ENTRY

### CODE REVIEW

**Comment by namannimmo10** [comment]
Please add "NO ENTRY" in the release notes. Like this:
```
<!-- BEGIN RELEASE NOTES -->
NO ENTRY
<!-- END RELEASE NOTES -->
```

**Comment by muskanvk** [comment]
> Please add "NO ENTRY" in the release notes. Like this:
> 
> ```
> 
> NO ENTRY
> 
> ```

Done thanks.

File: README.md [hunk 1]
```diff
@@ -184,7 +184,9 @@ Our issue tracker is at <https://github.com/sympy/sympy/issues>. Please
 report any bugs that you find. Or, even better, fork the repository on
 GitHub and create a pull request. We welcome all changes, big or small,
 and we will help you make the pull request if you are new to git (just
-ask on our mailing list or Gitter).
+ask on our [mailing list](https://groups.google.com/g/sympy?pli=1) or [Gitter Channel]
+(https://gitter.im/sympy/sympy). If you further have any queries, you can find answers
```
Comment (namannimmo10) [inline README.md:188]: Both the links are already there in the _README.md_ file

Comment (muskanvk) [reply README.md:188]: Saw just now. Thanks, but the stackoverflow part is not. I have added a link there too. 
Let me make changes and raise a PR again.

**Comment by czgdp1807** [comment]
+1

File: README.md [hunk 2]
```diff
@@ -184,7 +184,8 @@ Our issue tracker is at <https://github.com/sympy/sympy/issues>. Please
 report any bugs that you find. Or, even better, fork the repository on
 GitHub and create a pull request. We welcome all changes, big or small,
 and we will help you make the pull request if you are new to git (just
-ask on our mailing list or Gitter).
+ask on our mailing list or Gitter Channel. If you further have any queries, you can find answers
```
Comment (namannimmo10) [inline README.md:187]:
```suggestion
ask on our mailing list or Gitter Channel). If you further have any queries, you can find answers
```

Comment (namannimmo10) [reply README.md:187]: Just this one suggestion, otherwise, looks good to me.

**Comment by muskanvk** [comment]
Thanks for the help !!

**Review by namannimmo10 — APPROVED** [review]

**Pull request merged**